Entrega de 2 modulos y 1 extra

SaintRoguer opened the pull request:
Link a Heroku: https://trailerama.herokuapp.com/
Usuario : user@gmail.com 
Password : user
A consideracion, la ruta /favoritos solo se puede acceder si se esta logueado, ahora mismo da error pero es porque la parte de autorizacion y roles no esta realizada.
La barra de busqueda en la pagina inicial no esta funcionando, planeado hacerlo.

SaintRoguer committed:
Added Permissions library

added spatie for laravel

SaintRoguer committed:
Added Seeder for roles

Missing urls it's going to apply to.

SaintRoguer committed:
Added various tables and models

-art
-cast
-critic
-image
-tag

SaintRoguer committed:
Modified and added models and migrations

-cast
-comment
-actor_actress

SaintRoguer committed:
Factory and seed done

Looking to revamp comment system.

SaintRoguer committed:
Image factory doesnt work

Local permission problems

SaintRoguer committed:
Solved issue with images not downloading

vendor\fakerphp\faker\src\Faker\Provider\Image.php
curl_setopt($ch, CURLOPT_FILE, $fp);
            curl_setopt($ch, CURLOPT_SSL_VERIFYHOST, false); //nueva línea
            curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false); //nueva línea
            $success = curl_exec($ch) && curl_getinfo($ch, CURLINFO_HTTP_CODE) === 200;

SaintRoguer committed:
Fixed nav bar

SaintRoguer committed:
Art index and show almost done

index is done
show is missing:
-making the embeded youtube video work.
-add images for each actor/actress
-move the css stuff to common.css

SaintRoguer committed:
Created routes and modified image holder

Now actor/actress has image
Done all routes
Actor_actress:
-index
-create
-store
-edit
-update
-delete

SaintRoguer committed:
Added 2 new routes

Function to add or remove an actor/actress from a movie or tvshow done.

SaintRoguer committed:
Done critics

Routes+ambs

SaintRoguer committed:
Done with favorites and validators

Controlled all validators and favorites amb.

SaintRoguer committed:
Composer change

 "post-install-cmd": [
            "ln -sr storage/app/public public/storage"]
added to scripts section

SaintRoguer committed:
In process of getting inline classes onto .css file

Done with actors
addToArt
create
edit
index

SaintRoguer committed:
Done with css files

Done arts, critics & favorites views.

SaintRoguer commented:
Me olvide de decir que el boton "Add to list" que se encuentra en la vista /show de arts tambien necesita un usuario logueado, sino da error.

dcm-uns reviewed (changes requested):
- Revisar la accesibilidad - los mensajes de error en formularios son del mismo color que el fondo. Tambien con ciertos textos que hacen imposible leer la consigna.
- El formulario de carga de una pelicula tiene el mismo descriptor en todos los campos.
- La carga de una pelicula sin todos los campos completos tiró error SQL (ver issue)
- No fue posible editar una pelicula sin que se produzca un error SQL.

SaintRoguer committed:
Added Tags and fixed error

Added tag MVC and the many to many management
Fixed issues.

SaintRoguer committed:
Fixed issues & added new features

Fixed validators that made it impossible to create or edit a movie
Changed colour of letters to make them readeable.
Fixed Shrek issue.
Now when an user needs to be logged in it redirects to login screen, fixing error 500 issue on favorites/
Added Api functionality with optimization issues that i dont know how to fix, because image storing is costly.
Added Tags Crud

SaintRoguer committed:
Added file related to php.ini

Procfile modified so it recognizes user.ini
user.ini emulates local php.ini lines.

SaintRoguer committed:
Update ArtController.php

changed if author of ciritc is empty, its name becames the publisher

SaintRoguer committed:
Update ArtController.php

if exist it redirects to the show route.

SaintRoguer committed:
Published assets and changed icons

Jetstream assests and changed mark, auth and app icon.

SaintRoguer committed:
Added info tab

Us with info about the project.

SaintRoguer committed:
Update us.blade.php

SaintRoguer committed:
Update us.blade.php

SaintRoguer committed:
Fixed errors and deleted search

Fixed error in create tag button with wrong text, and deleted search bar and button in arts index.

SaintRoguer committed:
Fixed errors

Changed colour of text to be visible in dark background & fixed validate in favorite controller

SaintRoguer commented:
Abrire otro pr para agregar etiquetas
